CONTRIBUTING.md

```diff
@@ -2,116 +2,134 @@
 
 For adding a **pre-existing version of a protocol that was written in `docx` format**, follow the steps to create a new protocol, except in the second step:
 
-- a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
-- use the protocol-code from the pre-existing `docx` protocol to create a new branch
-- in case the chapter titles and Rmarkdown file names differ from the template, change them so they comply with the current template:
-    - go to [templates](https://github.com/inbo/protocolhelper/tree/master/inst/rmarkdown/templates)
-    - navigate to the template that you need and then navigate to the skeleton folder
-    - inspect the Rmarkdown file names and chapter titles
-    - the file `skeleton.Rmd` should be read as `index.Rmd`
-- continue the steps outlined for a new protocol.
+-   a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
+-   use the protocol-code from the pre-existing `docx` protocol to create a new branch
+-   If the section titles in the `docx` version of the protocol comply with section titles in the templates used by the `protocolhelper` package, than you will normally not see Rmarkdown file names starting with the same number. Otherwise, the function will have written both empty template Rmarkdown files as well as Rmarkdown files resulting from conversion of the `docx` file (Note that `NEWS.md` and `index.Rmd` are always created from the `protocolhelper` templates). In that case, you need to make changes ( renaming of files, deleting of redundant files) so that Rmarkdown file names comply with the [template Rmarkdown files](https://github.com/inbo/protocolhelper/tree/master/inst/rmarkdown/templates)
+-   continue the steps outlined for a new protocol.
 
 ## Starting a new protocol with the aid of protocolhelper functions
 
-The name and location of the protocol files and folder will be automatically determined by means of the input that you provide as arguments of the `create_`-family of functions (`create_protocol()`, `create_spp()`, `create_sfp()`). 
+The name and location of the protocol files and folder will be automatically determined by means of the input that you provide as arguments of the `create_`-family of functions (`create_protocol()`, `create_spp()`, `create_sfp()`).
 With `render = TRUE` the Rmarkdown files will be rendered to `html` output in a corresponding folder inside `docs`.
 This will allow you to check the resulting output locally.
 
-### From an existing docx protocol
+### From an existing docx protocol {#from-an-existing-docx-protocol}
 
-```r
+``` {.r}
```

Comment:
Fixed in 5dcde80. It was a side-effect of saving in canonical mode (see https://rstudio.github.io/visual-markdown-editing/#/markdown?id=markdown-output). Since, this is switched off, I have reverted this.